rpcs3/Emu/System.cpp

```diff
@@ -643,22 +642,28 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 		// Set RTM usage
 		g_use_rtm = utils::has_rtm() && ((utils::has_mpx() && g_cfg.core.enable_TSX == tsx_usage::enabled) || g_cfg.core.enable_TSX == tsx_usage::forced);
 
-		// Log some extra info in case of boot
 		if (!add_only)
 		{
+			// Log some extra info in case of boot
 #if defined(_WIN32) || defined(HAVE_VULKAN)
 			if (g_cfg.video.renderer == video_renderer::vulkan)
 			{
 				sys_log.notice("Vulkan SDK Revision: %d", VK_HEADER_VERSION);
 			}
 #endif
-
 			sys_log.notice("Used configuration:\n%s\n", g_cfg.to_string());
 
 			if (g_use_rtm && !utils::has_mpx())
 			{
 				sys_log.warning("TSX forced by User");
 			}
+
+			// Initialize patch engine
+			g_fxo->need<patch_engine>();
+
+			// Load patches from different locations
+			g_fxo->get<patch_engine>().append_global_patches();
+			g_fxo->get<patch_engine>().append_title_patches(m_title_id);
```

Comment:
I think these functions can be inserted inside the constructor.

Reply:
They can, but I think it's not worth the trouble in case someone accidentally uses it non-statically